flow_action_components/CollectionProcessors/force-app/main/default/classes/ListActionsTest.cls

```diff
@@ -5,22 +5,68 @@ public with sharing class ListActionsTest {
     private final static String TEST_CONTACT_NAME = 'Test Contact';
     private final static String FLOW_DATE_FORMAT = 'MMMM dd, yyyy';
     private final static String FLOW_DATE_TIME_FORMAT = 'MM/dd/yyyy, hh:mm aa';
+    // Counter to use as an offset when multiple account list retrievals are
+    // performed in one test
+    private static Integer numAccountsRetrieved = 0;
+
+    @TestSetup
+    static void makeData() {
```

Comment:
New `@TestSetup` method to minimize the amount of repeat DML across the test class.